src/Components/Tooltip/Tooltip.module.scss

```diff
@@ -0,0 +1,91 @@
+// Themes
+@import '../../constants/Themes/MainTheme';
+
+.tooltipContainer {
+  position: relative;
+  display: inline-block;
+  color: white;
+
+  .tooltipMsg {
+    background: $Color-TooltipBG;
+    border-radius: 5px;
+    border: solid 1px darkgray;
+    display: inline-block;
+    margin: 0;
+    padding: 4px 8px;
+    position: absolute;
+    text-align: left;
+    visibility: hidden;
+    width: max-content;
+
+    &.visible {
+      visibility: visible
+      ;
+    }
+
+    &::after {
+      content: " ";
+      border-width: 5px;
+      border-style: solid;
+      position: absolute;
+    }
+
+    &.top {
+      bottom: 130%;
+      left: 10%;
+
+      &::after {
+        border-color: black transparent transparent transparent;
```

Comment:
This is hard to read. Maybe you could create a var in theme file with a more descriptive name. Also usable in other cases if needed.

Reply:
Same for the other 3 cases.

Reply:
Done.